### DISCUSSION TITLE
Option to prevent horses from rearing when taking damage

### DISCUSSION THREAD

**Discussion by galacticwarrior9** [PurpurMC/Purpur#956] [__Completed__] [answered]
When horses take damage, they usually momentarily stop moving and rear (stand on their hind legs).

This is a major obstacle to improving mounted combat, and there does not appear to be a way to stop this from happening via the API. Even invoking `AbstractHorse#setRearing(false)` on a horse every tick does not eliminate this behaviour entirely - this also interferes with horse jumping.

The existing Purpur `stand-with-rider` option toggles whether horses will rear when their ambient sound is played. It would be great if there was a similar option for toggling this when a horse takes damage.

**Comment by katekerllenevich** [comment, marked as answer]
This option already exists! https://purpurmc.org/docs/Configuration/#stand-when-hurt

**Comment by galacticwarrior9** [comment]
Yep, I ended up PRing it

**Reply by katekerllenevich** [reply]
Ah I didn't even notice!

**Comment by galacticwarrior9** [comment]
I noticed this option isn't available anymore. Is there a reason for that? I can't find the commit removing it.
